src/sig/qtesla/CMakeLists.txt

```diff
@@ -1,3 +1,5 @@
+# SPDX-License-Identifier: MIT
```

Comment:
It looks you are only marking the files not in `external` directories, correct? qtesla doesn't use an external directory, but instead uses a `portable` and `avx2` one. Perhaps these should be moved down into an `external` folder for consistency, and not have the license applied to each external file.

Reply:
Yes, I marked every file not in an external or pqclean directory.